package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "todolist_review",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "dev": "webpack serve",
+    "test": "jest"
+  },
```

Comment:
- [ ] Let's ensure that the app can run correctly by adding the appropriate parameters to the _script_ section of the _package.json_, like the below:

```javascript
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack serve --mode development"
  },
```